src/widgets/menu-select/menu-select.tsx

```diff
@@ -92,7 +95,9 @@ const renderer = ({
   isFirstRendering: boolean
 ) => {
   if (isFirstRendering) {
-    renderState.templateProps = prepareTemplateProps({
+    renderState.templateProps = prepareTemplateProps<
+      MenuSelectComponentTemplates
```

Comment:
does this not get inferred from the arguments?

Reply:
It now does since we typed the `defaultTemplates`: https://github.com/algolia/instantsearch.js/pull/4785/commits/8f538d06fbe57af7d8ff0f6b73cecd88bb90997b